Tasks/MavenV3/utils.ts

```diff
@@ -0,0 +1,125 @@
+import * as path from 'path';
+import * as fs from 'fs';
+import * as tl from 'azure-pipelines-task-lib/task';
+import * as xml2js from 'xml2js';
+import * as shelljs from 'shelljs'
+
+export async function readFile(filePath: string, encoding: string) {
```

Comment:
Please add jsdoc comments for all new functions

Reply:
Done